### DISCUSSION TITLE
Blinking HDD codes - significance?

### DISCUSSION THREAD

**Discussion by mss71gfx** [PartialVolume/shredos.x86_64#124] [Q&A] [answered]
Hello, 

To start, I'm a newer user of ShredOS. I'm impressed with it and has helped a ton with wiping laptop drives via USB boot. Towards the end of my 12 laptops, I noticed that two had blinking ?codes? but seemed to progress just the same through the wiping procedure.

Both laptops had Seagate st500lm021-1kj152 drives. One flashed _[ 33C ]_, and the other was flashing _[ 21C ]_. After some time, I realized the numbers changed over time. I presume this has something to do with what sector it's operating on, and that even the laptops without flashing "codes" still change. The on screen location of this information is between the HDD size and the HDD model number.

Can you elaborate on the significance? Thanks!

**Comment by PartialVolume** [comment, marked as answer]
That's not a code, thats the drive temperature in degrees Celsius. Here's some further info on the meaning of a flashing temperature and also a change in color. https://github.com/PartialVolume/shredos.x86_64/releases/tag/v2020.05.017_x86-64_0.32.003

**Reply by PartialVolume** [reply]
If it was flashing at 21 degC I would suspect that the max, min and critical temperature values retrieved from the drive are not valid. Was it flashing black or red? Not that it matters too much. 21 deg.C is pretty cool for a drive, during a wipe on a spinning disc you would typically see that value rise from room temperature to 30-40 degC. Maybe hotter in a laptop with insufficient cooling and a high ambient room temperature.

**Reply by mss71gfx** [reply]
Thanks for the info! It was flashing red, but they completed successfully. They're being e-waste recycled, so I'm not too concerned about overheating.

I have 3 more running now with one flashing red. Once again, it's the same model Seagate drive. The highest I saw the other two was 44 C, (111 F - I'm in the states LOL). The one that's flashing now currently reads 40 C. We're in NY area, and the ambient room temp is probably only around 20 C or so with fair air circulation, but they were/are on a counter rather close side by side (apx 1/2 - 3/4" separation, small counterspace). So I wouldn't be surprised if they're griping about being too close and hot air from one feeding into the other etc. 

Regardless - thanks again. At least now if someone else looks up 'blinking' in the discussion thread they'll find more info ;)

**Reply by Firminator** [reply]
Showing wrong temperature is a common SMART issue based on faulty implementation of vendor and/or missing documenation. In ShredOS it's just a cosmetic issue.

**Reply by PartialVolume** [reply]
When nwipe version 0.35 is released there will be a change in the meaning of the text color of the temperature. The text will no longer flash.

Temperature text will be red on blue background if the drive temperature exceeds the drives reported high critical temperature.
Temperature text will be black on blue background if the drive temperature drops below the drives reported low critical temperature.

The drives temperature limits, assuming the drive reports them will be shown in nwipe's log which you can see when you exit nwipe.
